src/core/main.c

```diff
@@ -1364,7 +1363,6 @@ int main(int argc, char *argv[]) {
 
                 /* Running outside of a container as PID 1 */
                 arg_system = true;
-                make_null_stdio();
```

Comment:
by host case i mean --system mode on baremetal.

i wouldn't bother wir changing the defaults of stdout/stderr of services (and that includes user@.service) unless there's a very good reason to.